.github/workflows/root-ci.yml

```diff
@@ -384,13 +384,13 @@ jobs:
         with:
           name:  Test Results ${{ matrix.image }} ${{ matrix.config }}
           path: /tmp/workspace/build/TestResults.xml
-      - name: Setup tmate session
-        if: failure() && runner.debug == '1'
-        uses: mxschmitt/action-tmate@v3
-        with:
-          sudo: false
-        # When debugging increase to a suitable value!
-        timeout-minutes: ${{ github.event.pull_request && 40 || 20 }}
+      #- name: Setup tmate session
```

Comment:
I think you can just remove all the lines.

Reply:
Thanks Andre, done